### PR TITLE
add test for bank

### PR DESCRIPTION
**Pull request by emjay0921**

### CODE REVIEW

File: g2p_bank/tests/test_bank.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        if bank:
+            bank_id = bank[0]
+        else:
+            vals = {"name": "Sample Bank", "bic": "1010101010"}
+            bank_id = self.env["res.bank"].create(vals)
+
+        vals = []
+
+        # Generate Number Account Number
+        random_acc = str(random.randint(1, 9999999999))
```
Comment (jeremi) [inline g2p_bank/tests/test_bank.py]: Best to hardcode one.

File: g2p_bank/tests/test_bank.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+        # Compute Expected IBAN
+        expected_iban = IBAN.generate(
+            bank_id.country.code,
+            bank_code=bank_id.bic,
+            account_code=random_acc,
+        )
```
Comment (jeremi) [inline g2p_bank/tests/test_bank.py]: Do not use the same function as you are testing to generate the expected result. hardcode the expected result.

**Review by jeremi — CHANGES_REQUESTED** [review]

File: g2p_bank/tests/test_bank.py (hunk 1 above)
Comment (emjay0921) [reply g2p_bank/tests/test_bank.py]: Okay got it

File: g2p_bank/tests/test_bank.py (hunk 2 above)
Comment (emjay0921) [reply g2p_bank/tests/test_bank.py]: Okay

**Review by jeremi — APPROVED** [review]

**Pull request merged**